pkg/microservice/aslan/core/service/service/service.go

```diff
@@ -1000,7 +1001,7 @@ func updateGerritWebhookByService(lastService, currentService *commonmodels.Serv
 		log.Errorf("updateGerritWebhookByService getGerritWebhook err:%v", err)
 		//创建webhook
 		gerritWebhook := &gerrit.Webhook{
-			URL:       fmt.Sprintf("%s/%s%s", config.AslanAPIBase(), "gerritHook?name=", currentService.ServiceName),
+			URL:       fmt.Sprintf("%s?name=%s", config.WebHookURL(), currentService.ServiceName),
```

Comment:
Is the webhook API correct? It used to be /api/aslan/gerrithook?name= instead of /api/aslan/webhook?name=